protobuf/grpc_service.proto

```diff
@@ -908,31 +909,47 @@ message InferStatistics
   //@@  .. cpp:var:: StatisticDuration queue
   //@@
   //@@     The count and cumulative duration that inference requests wait in
-  //@@     scheduling or other queues.
+  //@@     scheduling or other queues. The "queue" count and cumulative 
+  //@@     duration includes cache hits.
   //@@
   StatisticDuration queue = 3;
 
   //@@  .. cpp:var:: StatisticDuration compute_input
   //@@
   //@@    The count and cumulative duration to prepare input tensor data as
   //@@    required by the model framework / backend. For example, this duration
-  //@@    should include the time to copy input tensor data to the GPU.
+  //@@    should include the time to copy input tensor data to the GPU. The
+  //@@    "compute_input" count and cumulative duration DO NOT include cache
```

Comment:
Besides just saying that it does not include cache hits, can we add an additional line explaining why they don't account for requests that hit the cache? Can be as simple as saying:
```
The "compute_input" count and cumulative duration do not account for requests that were a cache-hit. When using response cache, preparing input tensor data for model/backend consumption is not required for requests that hit cache. <and link caching document here>  
```